.github/workflows/test-build.yml

```diff
@@ -42,6 +42,7 @@ jobs:
       fail-fast: false
       matrix:
         host_release:
+        - unstable
```

Comment:
Is it worth squashing this into commit 14b94d049c9c7c545754c2d6b5709cd218891c55?

Reply:
squashed